apps/digital-inclusion/behaviours/save-data.js

```diff
@@ -1,55 +1,25 @@
 'use strict';
 
-const mysql = require('mysql');
-const config = require('../../../config');
+// const mysql = require('mysql');
+// const config = require('../../../config');
 
-const pool = mysql.createPool({
-  host: config.mysql.host,
-  user: config.mysql.user,
-  password: config.mysql.password,
-  database: config.mysql.db
-});
+// const pool = mysql.createPool({
+//   host: config.mysql.host,
+//   user: config.mysql.user,
+//   password: config.mysql.password,
+//   database: config.mysql.db
+// });
 
-const table = 'research_data';
```

Comment:
Delete rather than comment out. You can always look at old commits to see the old code.

Reply:
I agree, I just forgot to delete it.  I'll do that now